site/en/docs/extensions/mv3/webusb/index.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: "layouts/doc-post.njk"
+title: "WebUSB in extensions"
+seoTitle: "Chrome Extensions: WebUSB"
+date: 2023-11-02
+description: The WebUSB API, which exposes non-standard Universal Serial Bus (USB) compatible devices to the web, is available in extensions.
+---
+
+The WebUSB API exposes non-standard Universal Serial Bus (USB) compatible devices to the web. This page describes aspects of the API that are particular to extensions. Refer to MDN for complete details of the [WebUSB API](https://developer.mozilla.org/docs/Web/API/WebUSB_API).
+
+Here is [a sample app for WebUSB](https://github.com/sowbug/weblight/tree/master);
+
+## Availability in extensions
+
+Chrome 117 or later.
```

Comment:
According to https://chromestatus.com/feature/5200265459269632, WebUSB support is Chrome 118.

Reply:
Fixed.